examples/Dancing_with_Color/3D_Rotation_Light_Up/3D_Rotation_Light_Up.ino

```diff
@@ -21,7 +21,7 @@ void setup() {
 }
 
 // Now, we will need to initialize three variables, 
-// for the red, green, and blue LED of each big 
+// for the red, green, and blue component of each  
```

Comment:
Since both the large and small LEDs are turned on for this activity, this should say of each LED and not just the large LEDs.